snippets/csharp/System.Linq/Enumerable/AggregateTSource/enumerable.cs

```diff
@@ -338,9 +331,9 @@ static void AverageEx2()
         static void AverageEx3()
         {
             // <Snippet16>
-            string[] numbers = { "10007", "37", "299846234235" };
+            string[] numbers = ["10007", "37", "299846234235"];
 
-            double average = numbers.Average(num => long.Parse(num));
+            double average = numbers.Average(long.Parse);

```

Comment:
I'm ambivalent on this change. Less experienced programmers may be confused by using a method group instead of a lambda expression. If we keep this change, we should add a note in the corresponding text to explain a method group conversion.